src/pages/template.tsx

```diff
@@ -0,0 +1,230 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { ArrowDown, Mail, AlertCircle, HelpCircle } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import * as React from "react";
+
+export function ContactButton() {
+  return (
+    <div className="button-container">
+      <Button variant="contactButton1" size="contactButtonSize">
```

Comment:
@rakeem996 This component is not re-useable as it stands because if this sample component is used in different pages and you make a change in the template page, it will not change everywhere. You would have to go to all 20 pages and make the change.

The goal here is to make it easier to change the styles app wide without making a change in every place so you need to create a component called button that accepts a prop called `variant`. Those can be 
1. primary
2. secondary
3. outlined
4. text

From here https://www.figma.com/design/8x4CwEJ8ESDjPMPNJLaQ5k/Octalogic-Tech-%7C-Website-Design?node-id=4-17567&t=AxRdN85kltW907Ex-1